README.MD

```diff
@@ -35,7 +36,30 @@ module: {
     loaders: [
         {
             test: /\.js$/,
-            loader: WebPackAngularTranslate.jsLoader()
+            loader: WebPackAngularTranslate.jsLoader(),
+            query: {
+                sourceType: 'script', // acorn option default
+                allowImportExportEverywhere: false // acorn option default
```

Comment:
Either way that was just the default option, but I needed to use it for some reason. But I'll remove this from the example either way. `sourceType: 'script'` is also the default. The defaults are listed [here](https://github.com/ternjs/acorn/blob/master/src/options.js)